guides/gatherers.md

```diff
@@ -1575,3 +1577,119 @@ Example output (in Rhai):
 // hacluster_has_default_password
 true;
 ```
+
+<span id="ini_files"></span>
+
+### ini_files@v1
+
+**Argument required**: yes.
+
+This gatherer fetches the content from a configuration file in INI format. The configuration file is specified as argument, choosen from a list of allowed files.
+Currently whitelisted files are:
+
+- `global.ini`
+
+Each fact request can return one or more item, one for each found file; multiple files can occur when the host has configured more than one SAP system. Each item then has a `sid` field with the system id and a `value` field with the actual content of the file.
+
+Example arguments:
+
+| Name         | Return value                                                                         |
+| :----------- | :----------------------------------------------------------------------------------- |
+| `global.ini` | Retrieved the content from `/usr/sap/<sid>>/SYS/global/hdb/custom/config/global.ini` |
+
+```yaml
+facts:
+  - name: global_configuration
+    gatherer: ini_files@v1
+    argument: global.ini
+```
+
+Example output (in Rhai):
+
+```ts
+[
+  #{
+    "sid": "S01",
+    "value": #{
+      "communication": #{
+        "internal_network": "10.23.1.128/26",
+        "listeninterface": ".internal"
+      },
+      "internal_hostname_resolution": #{
+        "10.23.1.132": "hana-s1-db1",
+        "10.23.1.133": "hana-s1-db2",
+        "10.23.1.134": "hana-s1-db3"
+      }
+    }
+  }
+]
+```
+
+<span id="sudoers"></span>
+
+### sudoers@v1
+
+**Argument required**: no.
+
+This gatherer fetches the sudoer information about a user. The output is a list of objects representing the sudoer rules with the following fields:
+
+* `user`: The name of the user the rule applies to;
+* `command`: The command a sudoer rule has been specified for;
+* `run_as_user`: The user privilege the command will be executed with;
+* `run_as_group`: The group privilege the command will be executed with;
+* `no_password`: Whether the `NOPASSWD` tag is set for the rule.
+
+The gatherer operates in two modes:
+
+* _explicit user mode_: the target user is specified as the gatherer argument;
+* _user discovery mode_: no argument is specified, thus the gatherer fetches results for all the configured users for the SAP systems on the host.
+
```

Comment:
nitpick: maybe we want to add the `Example arguments` table. I know it is explained above, but...

Reply:
Done in https://github.com/trento-project/wanda/pull/588/commits/8675adb9b0941725c1e49baa38dd4422b9c5266b